integration/test_aggregation_integration.py

```diff
@@ -976,8 +976,8 @@ def test_query_aggregate_by_key_fixed_window_new_time_exceeds_stored_window(setu
     actual = controller.await_termination()
     expected_results = [
         {'col1': 0, 'number_of_stuff_count_30m': 1, 'number_of_stuff_count_2h': 1},
-        {'col1': 1, 'number_of_stuff_count_30m': 1, 'number_of_stuff_count_2h': 1},
-        {'col1': 2, 'number_of_stuff_count_30m': 1, 'number_of_stuff_count_2h': 2},
+        {'col1': 1, 'number_of_stuff_count_30m': 1, 'number_of_stuff_count_2h': 2},
+        {'col1': 2, 'number_of_stuff_count_30m': 1, 'number_of_stuff_count_2h': 3},
```

Comment:
Thing is, the original expected results look right to me... What's the logic behind the new expected results?